prism-agent/service/server/src/test/resources/logback.xml

```diff
@@ -12,6 +12,7 @@
 
     <logger name="io.getquill" level="OFF"/>
     <logger name="com.zaxxer.hikari" level="OFF"/>
+    <logger name="org.jboss.resteasy.client" level="OFF"/>
```

Comment:
does it make sense to keep the ERROR level, or do we wrap up the error and log it at our service layer?

Reply:
I could change it to ERROR level as well. It's just annoying WARN log in the test that we need to silence.